src/store/useStore.js

```diff
@@ -6,6 +6,7 @@ import createRepositorySlice from './RepositorySlice'
 
 
 const useStore = create((set, get) => ({
+  widgetApiUri: null,
```

Comment:
I suspect this will grow quickly.  Please make a slice for it called ApiSlice with field URI